tools/cli/app.go

```diff
@@ -84,9 +84,9 @@ func NewCliApp() *cli.App {
 			EnvVar: "TEMPORAL_CLI_TLS_CA",
 		},
 		cli.BoolFlag{
-			Name:   FlagTLSEnableHostVerification,
-			Usage:  "validates hostname of temporal cluster against server certificate",
-			EnvVar: "TEMPORAL_CLI_TLS_ENABLE_HOST_VERIFICATION",
+			Name:   FlagTLSDisableHostVerification,
+			Usage:  "whether to validates hostname of temporal cluster against server certificate",
```

Comment:
same nit here: maybe word as - "disables validation of the temporal cluster's server certificate"